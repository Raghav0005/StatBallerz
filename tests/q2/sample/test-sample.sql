SELECT g.GameID, stats.avg_points
FROM Games g
JOIN (
    SELECT GameID AS gid, AVG(Points) AS avg_points
    FROM PlayedIn
    GROUP BY GameID
) stats ON g.GameID = stats.gid
WHERE g.GameDate >= '2024-10-25' AND g.GameDate <= '2024-10-27';