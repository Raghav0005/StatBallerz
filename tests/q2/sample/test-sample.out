
GAMEID      AVG_POINTS 
----------- -----------
       1001          29
       1002          28
       1003          26

  3 record(s) selected.


